Learning/eval_policy.py

```diff
@@ -281,15 +402,26 @@ def save_data(metrics, trial, save_dir):
         # vis = Matplotlib_Visualizer(env.wrapper_env)
         # vis = OpenCV_Visualizer(env.wrapper_env)
         vis = Pillow_Visualizer(env.wrapper_env)
-        if args.policy == 'b':
+        if args.policy == 'ba':
             if args.multi:
                 env = init_env(rows, cols, depth, sector_rows, sector_cols, prune_window_rows, prune_window_cols, action_low,
                     action_high, obs_low, obs_high, collection_time_steps, garden_step, num_plant_types, seed, args.multi)
-                evaluate_baseline_policy_multi(env, baseline_policy.policy, collection_time_steps, sector_rows, sector_cols,
+                evaluate_analytic_policy_multi(env, analytic_policy.policy, collection_time_steps, sector_rows, sector_cols,
                                         prune_window_rows, prune_window_cols, garden_step, water_threshold,
                                         sector_obs_per_day, trial)
             else:
-                evaluate_baseline_policy_serial(env, baseline_policy.policy, collection_time_steps, sector_rows, sector_cols,
+                evaluate_analytic_policy_serial(False, env, analytic_policy.policy, collection_time_steps, sector_rows, sector_cols,
+                                        prune_window_rows, prune_window_cols, garden_step, water_threshold,
+                                        sector_obs_per_day, trial, save_dir, vis_identifier)
+        elif args.policy == 'bw':
+            if args.multi:
+                env = init_env(rows, cols, depth, sector_rows, sector_cols, prune_window_rows, prune_window_cols, action_low,
+                    action_high, obs_low, obs_high, collection_time_steps, garden_step, num_plant_types, seed, args.multi)
+                evaluate_analytic_policy_multi(env, analytic_policy.policy, collection_time_steps, sector_rows, sector_cols,
+                                        prune_window_rows, prune_window_cols, garden_step, water_threshold,
+                                        sector_obs_per_day, trial)
+            else:
+                evaluate_analytic_policy_serial(True, env, analytic_policy.policy, collection_time_steps, sector_rows, sector_cols,
```

Comment:
Make sure to change the position of "True" and "False" after changing the method signature of evaluate_analytic_policy_serial as suggested above.